dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -34,6 +34,16 @@ export JQ_BINARY_DOWNLOAD_URL=${JQ_BINARY_DOWNLOAD_URL:-${DEFAULT_JQ_BINARY_DOWN
 DEFAULT_CHE_MULTIUSER="false"
 export CHE_MULTIUSER=${CHE_MULTIUSER:-${DEFAULT_CHE_MULTIUSER}}
 
+DEFAULT_CHE_DEPLOY=false
+export CHE_DEPLOY=${CHE_DEPLOY:-${DEFAULT_CHE_DEPLOY}}
+CHE_DEPLOYED=false
+
+DEFAULT_CHE_REMOVE_PROJECT=false
+export CHE_REMOVE_PROJECT=${CHE_REMOVE_PROJECT:-${DEFAULT_CHE_MULTIUSER}}
```

Comment:
Are you sure that `DEFAULT_CHE_MULTIUSER` should be here as default value?

Reply:
Will revise. Thanks.